src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,77 @@
+export default function Footer() {
+
+  // this array of objects my have to be moved in App.jsx instead and passed through with props.. depending further development and if needed for other components
+  const airQualityLevels = [
```

Comment:
can you move data into seperate json or data files, please. it will be clean and more readable